server/mlabns/util/util.py

```diff
@@ -1,12 +1,15 @@
 import json
+import os
 
 import jinja2
 
 from mlabns.util import message
 
 def _get_jinja_environment():
+    current_dir = os.path.dirname(__file__)
```

Comment:
Yeah, I'd rather not use `__file__` but I don't know of any other way we can find relative paths. We can't use `getcwd()` because it depends on where the user started the unit tests from, not where the files are relative to each other:

```
mtlynch@mtlynch0:/tmp/cwd-test/a$ python test.py 
os.path.join(os.getcwd(), '../b')		         /tmp/cwd-test/b
os.path.join(os.path.dirname(__file__), '../b')	 /tmp/cwd-test/b
mtlynch@mtlynch0:/tmp/cwd-test/a$ cd ..
mtlynch@mtlynch0:/tmp/cwd-test$ python a/test.py 
os.path.join(os.getcwd(), '../b')		         /tmp/b           <<< BAD
os.path.join(os.path.dirname(__file__), '../b')	 /tmp/cwd-test/b
```

FWIW the GAE docs example uses `__file__`